rpc/README.md

```diff
@@ -6923,3 +7027,18 @@ The uncle block template of the new block for miners.
 The simple increasing integer version.
 
 This is a 32-bit unsigned integer type encoded as the 0x-prefixed hex string in JSON. See examples of [Uint32](#type-uint32).
+
+### Type `WitnessesProof`
+
+Merkle proof for transactions’ witnesses in a block.
+
+#### Fields
+
+`WitnessesProof` is a JSON object with the following fields.
+
+*   `block_hash`: [`H256`](#type-h256) - Block hash
+
+*   `witnesses_root`: [`H256`](#type-h256) - Merkle root of all transactions’ witness hash
+
+*   `witnesses_proof`: [`MerkleProof`](#type-merkleproof) - Merkle proof of transactions’ witnesses
```

Comment:
we need to provide the `raw_transactions_root` and verify the `block.transactions_root() == merkle_root(&[raw_transactions_root, witnesses_root])` in the `verify_witnesses_proof` rpc implemention.

```
WitnessProof {
  block_hash: H256
  raw_transactions_root: H256
  proof: MerkleProof
}
```

and suggest renmae the rpc name to `get_witness_proof` to be consistent with the `get_transaction_proof` rpc